sdk/search/Azure.Search.Documents/tests/SearchIndexClientTests.cs

```diff
@@ -18,7 +18,7 @@ namespace Azure.Search.Documents.Tests
     public class SearchIndexClientTests : SearchTestBase
     {
         public SearchIndexClientTests(bool async, SearchClientOptions.ServiceVersion serviceVersion)
-            : base(async, serviceVersion, null /* RecordedTestMode.Record /* to re-record */)
+            : base(async, serviceVersion, RecordedTestMode.Record /* to re-record */)
```

Comment:
Did you forget to revert that or is that deliberate?

Reply:
Nope. Good catch (though it seems CI caught it pretty quickly, too).